backend/apps/web/models/chats.py

```diff
@@ -20,6 +20,7 @@ class Chat(Model):
     title = CharField()
     chat = TextField()  # Save Chat JSON as Text
     timestamp = DateField()
+    share_id = CharField(null=True, unique=True)
```

Comment:
Fine with current implementation, but I'm thinking it might be bettter to have `public` or `shared` boolean flag and reuse `chat_id`?

Reply:
We might also want to add migration script somewhere that runs before the app starts to ensure backward compatibility. Ideally it should automatically detect added fields and reflect changes in our `webui.db`.

Reply:
Related: https://github.com/open-webui/open-webui/pull/1306